packages/i18n-utils/src/localize-url.tsx

```diff
@@ -98,7 +98,17 @@ const urlLocalizationMapping: UrlLocalizationMapping = {
 		url.pathname = url.pathname.replace( /\/help\//, '/support/' );
 		return prefixLocalizedUrlPath( supportSiteLocales )( url, localeSlug );
 	},
-	'wordpress.com': prefixLocalizedUrlPath( magnificentNonEnLocales ),
+	'wordpress.com': ( url: URL, localeSlug: Locale ) => {
+		// Don't rewrite checkout and me URLs.
+		if ( /^\/(checkout|me)(\/|$)/.test( url.pathname ) ) {
+			return url;
+		}
+		// Don't rewrite Calypso URLs that have the URL at the end.
```

Comment:
Is it supposed to match only the pathnames that have the url as last parameter, because the regex is currently matching no matter if the url is last parameter or not?

Reply:
Not sure I 100% unstand the question. This is to match Calypso style URLs that look like mentioned in the unit tests: https://wordpress.com/home/test.wordpress.com, https://wordpress.com/domains/test.wordpress.com etc. One edge case that is currently not rewritten and might need a change would be a URL like https://wordpress.com/test.wordpress.com, do you mean that?

Reply:
Ah, now I understand the mistake in my Regex, I'll update it.

Reply:
Updated. Thanks for catching this!